projects/realt.js

```diff
@@ -0,0 +1,94 @@
+const sdk = require("@defillama/sdk")
+const retry = require('./helper/retry')
+const axios = require("axios")
+const BigNumber = require("bignumber.js")
+
+// Get supply for token (from xdai or mainnet)
+async function getSupply(token, contractChain) {
```

Comment:
seems like this is not used, will merge and delete

Reply:
True, it was present in an older version.